backend/app/rest/invite_users_routes.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
is this file needed if the invited_users table is being removed?

Reply:
nope not needed, all the invited_users stuff will be deleted (service, routes, table, DTO)